lib/gui/models/store.js

```diff
@@ -95,8 +95,8 @@ const storeReducer = (state, action) => {
 
         const drive = _.first(action.data);
 
-        // Even if there's no image selectected, we need to call `isDriveValid`
-        // to check if the drive is locked, and `{}` works fine with it
+        // Even if there's no image selectected, we need to call several
```

Comment:
Fixed. I wonder if there exists a tool to check grammar in comments. Have you heard about such a tool?